src/api/user/auth.js

```diff
@@ -97,3 +108,20 @@ exports.askForgotPassword = wrapper(async (req, res, next) => {
   sendMail(mailBody)
   res.json('Check your email')
 })
+
+exports.getCurrentUser = wrapper(async (req, res, next) => {
+  const result = await verifyJwt(req.query.token, secret)
+  const user = await User.findOne({ where: { email: result.email } })
+
+  res.json(deleteUnnecessary((user)))
```

Comment:
specify which fields to be excluded in `findOne` rather then exclude by yourself